src/cswinrt/code_writers.h

```diff
@@ -3767,18 +3772,29 @@ evt.Name());
     {
         w.write(R"(
 private volatile static global::System.Runtime.CompilerServices.ConditionalWeakTable<object, EventSource<%>> _%_;
-private static global::System.Runtime.CompilerServices.ConditionalWeakTable<object, EventSource<%>> Make%Table()
+private static global::System.Runtime.CompilerServices.ConditionalWeakTable<object, EventSource<%>> _%
 {
-    %
-    global::System.Threading.Interlocked.CompareExchange(ref _%_, new(), null);
-    return _%_;
+    get
+    {
+        var table = _%_;
+        if (table != null)
+        {
+            return table;
+        }
+        else
+        {
+            %
+            global::System.Threading.Interlocked.CompareExchange(ref _%_, new(), null);
+            return _%_;
+        }
```

Comment:
Drop the `else` in all the properties (as there's a return before), just remove the indentation instead (leave a blank line)
```suggestion
        
        %
        global::System.Threading.Interlocked.CompareExchange(ref _%_, new(), null);
        return _%_;
```